add some requirements versions

mmurphy opened the pull request:
specify minimum versions of oc CLI and Homebrew
specify Go paths using the GOPATH envvar
use 'go get...' command instead of 'git clone...' to clone into $GOPATH
Add brackets () around the ansible installer command so that you are not left in the installer directory if the playbook fails

mmurphy commented:
The Homebrew version I specified is recent (from the last few days) so it might be a bit excessive.
The install on my mac is still not working, so there may still be some version issues

maleck13 commented on docs/walkthroughs/local-setup.adoc:
```diff
@@ -41,18 +42,18 @@ Clone this repository
 [source,bash]
 ----
-git clone git@github.com:feedhenry/mcp-standalone.git
+go get github.com/feedhenry/mcp-standalone
```
You only need go installed if you are going to do development, so would rather leave it as git clone in this doc

mmurphy replied:
ok no problem, that's not clear from the original doc though, I'll update this PR to clarify it

david-martin replied:
:+1:  on updating the doc to make it clearer

david-martin commented on docs/walkthroughs/local-setup.adoc:
```diff
@@ -15,8 +15,9 @@ Requirements
-* https://github.com/openshift/origin/releases[oc command line client]
+* https://github.com/openshift/origin/releases[oc command line client] >= 3.7.0-rc0
 * https://nodejs.org/en/[Node.js] >= 6.10.0
+* https://brew.sh[Homebrew] >= 1.3.6
```
Maybe add an "(Only required for Mac)" ?

mmurphy commented:
I've replaced the `git clone` commands and removed `go get`
I think it still needs a branch in the instructions for using go and not using go, since the current (and previous) instructions require that the user is familiar with the go directory structure, but that would be better as a separate PR.

david-martin commented on docs/walkthroughs/local-setup.adoc:
```diff
@@ -17,17 +17,18 @@ for more detailed guides.
 * https://hub.docker.com/[Docker Hub account]
 * https://docs.docker.com/engine/installation/[Docker]
-* https://github.com/openshift/origin/releases[oc command line client]
+* https://github.com/openshift/origin/releases[oc command line client] >= 3.7.0-rc0
```
I'd even remove this requirement as the installer will pull down the correct version of oc and put it in /usr/local/bin. This sound right to you @aidenkeating? Would you agree?

aidenkeating replied:
@david-martin Yep that's correct

mmurphy replied:
Ok, I understand how it's supposed to work, so this can be removed.
(However, I'll add an issue where the installer will run an existing older `oc` instead of the downloaded one, if it appears earlier in the PATH)

mmurphy replied:
Just thinking, since this may replace and existing version of `oc` it should at least warn the user beforehand

david-martin replied:
@mmurphy Great point! If you're creating that issue, you can put a link to it in this PR.
I'll remove that bullet from the doc and merge this PR

david-martin commented:
@mmurphy I've pushed a few changes, if you could take a look
b33969a

david-martin commented:
Thanks @mmurphy :+1:
